examples/ConfigurableFirmata/ConfigurableFirmata.ino

```diff
@@ -175,8 +175,11 @@ I2CFirmata i2c;
 #include <OneWireFirmata.h>
 OneWireFirmata oneWire;
 
-#include <StepperFirmata.h>
-StepperFirmata stepper;
+#include <StepperFirmata2.h>
+StepperFirmata2 stepper;
+
+#include <StepperFirmata2.h>
```

Comment:
Because I wanted to make damn sure it got included...

or I messed up a rebase somewhere...

Ya, it's probably that second one.